sdk/tests/core/test_az_mqtt5_connection.c

```diff
@@ -231,15 +232,16 @@ AZ_INLINE void _az_mqtt5_connection_setup(
     _az_hfsm* mock_client_hfsm_2)
 {
   *test_conn_options = az_mqtt5_connection_options_default();
-  test_conn_options->client_certificate_count = 2;
```

Comment:
Should it be `client_certificates_count` ? With S

Reply:
Just an idea. Usually "_count" is just suffixed into an existing array name for better reference.

Reply:
That's a quick change and it makes more sense, changed!